libraries/BLE/src/BLEEddystoneTLM.cpp

```diff
@@ -46,10 +44,14 @@ uint16_t BLEEddystoneTLM::getVolt() {
 	return ENDIAN_CHANGE_U16(m_eddystoneData.volt);
 } // getVolt
 
-float BLEEddystoneTLM::getTemp() {
-	return ENDIAN_CHANGE_U16(m_eddystoneData.temp) / 256.0f;
+uint16_t BLEEddystoneTLM::getTemp() {
+	return ENDIAN_CHANGE_U16(m_eddystoneData.temp);
 } // getTemp
```

Comment:
As said, this is a breaking change. I think it should stay the same:
`float BLEEddystoneTLM::getTemp()`

The original code works correctly.

Reply:
Suggestion is to eliminate `uint16_t BLEEddystoneTLM::getTemp()` and just keep original `float BLEEddystoneTLM::getTemp()` that converts U16 BigEndian to Float (return value) using the MACRO.

Reply:
This should be the final code:
``` cpp
float BLEEddystoneTLM::getTemp() {
	return EDDYSTONE_TEMP_U16_TO_FLOAT(m_eddystoneData.temp);
} // getTemp
```